src/cascadia/TerminalApp/Pane.h

```diff
@@ -72,6 +72,9 @@ class Pane : public std::enable_shared_from_this<Pane>
 
     int GetLeafPaneCount() const noexcept;
 
+    void Zoom(std::shared_ptr<Pane> zoomedPane);

```

Comment:
Now that I think about it, if we took MDI parlance... it might be `Maximize` and `Restore` here... but that's probably more confusing than `Zoom` and `UnZoom`.